scripts/svm/simpleFill.ts

```diff
@@ -36,7 +35,7 @@ const argv = yargs(hideBin(process.argv))
   .option("inputAmount", { type: "number", demandOption: true, describe: "Input amount" })
   .option("outputAmount", { type: "number", demandOption: true, describe: "Output amount" })
   .option("originChainId", { type: "string", demandOption: true, describe: "Origin chain ID" })
-  .option("depositId", { type: "string", demandOption: true, describe: "Deposit ID" })
+  .option("depositId", { type: "array", demandOption: true, describe: "Deposit ID" })
```

Comment:
its too cumbersome to pass array in command line (need to repeat --depositId param for each byte. Better use string and parse it as hex (if it starts with 0x) or number